### DISCUSSION TITLE
This is a great piece of work

### DISCUSSION THREAD

**Discussion by go-linux** [immense/Remotely#425] [General]
First i like to thank you, for your great work.

second, i had a problem installing the software on a debian 11 vm with nginx
it stopped with the error (not the exact error) after downloading starting the installer

"no such file or directory"
after digging it seems to search for Ubuntu details - which are not available
after changing the  Ubuntu_nginx_install.sh i
added after line 32
-- code
# Install .NET Core Runtime.
# check if Ubuntu or debian

FLAVOR=$(lsb_release -i -s)
Version=$(lsb_release -r -s)

if [[ ${FLAVOR} == "Ubuntu" ]];then
   wget -q https://packages.microsoft.com/config/ubuntu/$Version/packages-microsoft-prod.deb
   dpkg -i packages-microsoft-prod.deb
   add-apt-repository universe
   elif [[ ${FLAVOR} == "Debian" ]];then
      echo "it's a debian system"
      wget -q https://packages.microsoft.com/debian/$Version/prod/pool/main/p/packages-microsoft-prod/packages-microsoft-prod_1.0-debian11.1_amd64.deb
      dpkg -i packages-microsoft-prod.deb
   else
      echo "no supported system recognized, bye"
      exit 1
fi
-- end code

this way i added ms.list manually to /etc/apt/sources.list.d 
echo "deb [arch=amd64,arm64,armhf] https://packages.microsoft.com/debian/11/prod bullseye main" > /etc/apt/sources.list.d/ms.list

after running the script manually its running through

it's not nice but it works

hope it helps if someone has the similar problem

**Comment by stevesobol** [comment]
That's awesome. Would you consider putting this change into a pull request?
